Island.java

```diff
@@ -1,16 +1,76 @@
 import java.util.*;
 
+
 /**
  * An island is part of a FIRE resort.Each island has a name,  a luxury rating
  * and a capacity which represents the maximum number of people(passes) who can be on the
  * island at any one time. Each island must maintain a list of all people (passes)
  * currently on the island. These lists are updated whenever passes enter or leave
  * an island,so that it is always possible to say which passes are on the island
  *
- * @author (your name)
+ * @author (20067052)
  * @version (a version number or a date)
  */
-
 public class Island {
+    private int island_ID;
```

Comment:
Same as earlier, need to be in `camelCase`.

Reply:
done